### ISSUE TITLE
License issue, pecl?

### ISSUE THREAD

**Issue by pierrejoye** [chobie/php-uv#55]
hi,

The PHP License cannot be used for anything not released under *.php.net. An easy way to solve this problem is to release this extension under pecl.php.net.

Pros for this move besides solving the license issue:
- more visibility
- automatic windows builds
- more peer reviews
- works with pickle install on all platforms :)

Thanks for your work!

**Comment by chobie** [comment]
@pierrejoye Thanks pointing the issue. I'll consider to release this extension under pecl.
May I keep PHP license while blushing up this?

**Comment by pierrejoye** [comment]
yes, once you release it there (repo can remain on github), the PHP License
can be used :)

On Wed, Jul 23, 2014 at 7:04 AM, Shuhei Tanuma notifications@github.com
wrote:

> @pierrejoye https://github.com/pierrejoye Thanks pointing the issue.
> I'll consider to release this extension under pecl.
> May I keep PHP license while blushing up this?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/chobie/php-uv/issues/55#issuecomment-49833310.

## 

Pierre

@pierrejoye | http://www.libgd.org

**Comment by datibbaw** [comment]
I think now with 5817bfe4b04b6179f4d377f059e67a1b2d2e2e78 in place we can probably talk about this again.

IMO we can release the current procedural as a first version.

I'll be glad to help out if you need any help :smile_cat:

**Comment by chobie** [comment]
@datibbaw Good to hear :) I'll send an email to PECL mailing list in a few days.

**Comment by sasezaki** [comment]
@chobie any progress ? :wink:

**Comment by chobie** [comment]
Sorry about the delay. I was overwhelmingly busy these months as I had a tech event yesterday.
I'll re-schedule my oss related tasks in this weekend. I won't take long time to propose this to PECL.
